src/main/java/com/raoulvdberge/refinedstorage/apiimpl/storage/cache/StorageCacheFluid.java

```diff
@@ -56,9 +56,11 @@ public synchronized void invalidate() {
                 continue;
             }
 
-            for (StackListEntry<FluidStack> stack : storage.getEntries()) {
-                if (stack != null && stack.getCount() > 0)
-                    add(stack.getStack(), stack.getCount(), true);
+            for (StackListEntry<FluidStack> entry : storage.getEntries()) {
+                if (entry != null && entry.getCount() > 0) {
```

Comment:
The entries themselves can't be `null`, no need to check this. Seems like this was left here from previous refactoring.